models/shop.model.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
```

Comment:
this file seems to come from generation, it's not needed

Reply:
Mongo is used to store shop access token

Reply:
Let's use MYSQL for storing the token then

Reply:
Done it. Let's check